src/test/suite/toc.test.ts

```diff
@@ -292,6 +292,77 @@ suite("TOC.", () => {
             new Selection(0, 0, 0, 0)).then(done, done);
     });
 
+    test("Exclude omitted headings (`toc.omittedFromToc`)", (done) => {
```

Comment:
I added a (passing) test case but on my setup, 21 other tests currently fail on `master`. :thinking:

Reply:
There are chances that some tests fail because of timeout. I just re-ran the tests and it looks good now.

Reply:
Actually, it looks like some settings are overridden by my VSCode config.

For example `"editor.tabSize": 4`:

```diff
2) No list editing.
    Shift tab key:
    '    text':

    AssertionError [ERR_ASSERTION]: Expected values to be loosely deep-equal:
    '  text' should loosely deep-equal 'text'

+ expected
- actual

-  text
+text
```

Would it be possible to run the tests on a "vanilla" VSCode instance to prevent that?

Reply:
I started VSCode with `code --disable-extensions --user-data-dir /tmp vscode-markdown` and all tests are passing, now. :+1:
It would be nice to add these flags automatically to open the test's VSCode instance.

Reply:
Sounds like a good idea. I tried to add `--user-data-dir /tmp` to

https://github.com/yzhang-gh/vscode-markdown/blob/9c3438fe1fdc5ab9040316921c839d44c21a3596/.vscode/launch.json#L26-L30

but it doesn't seem to work 🤔

Reply:
Maybe `--user-data-dir=/tmp` would do the trick?
I'm not sure `/tmp` is available on Windows, however.

Reply:
You are right 👍